### ISSUE TITLE
Have a first-run link list pointing to useful materials

### ISSUE THREAD

**Issue by Daenyth** [Cockatrice/Cockatrice#4383]
The list could include things like: how to play, formats, ban list, deck builder, discord, glossary of terms and etiquette

**Comment by tooomm** [comment]
👍

I still think https://github.com/Cockatrice/Cockatrice/issues/3063 would be a very user friendly way to implement something like this.
Users see it prominently on first runs, always have it at hand, learn at their own pace and close it when they feel comfortable in doing so. Of course they can always come back to it just like any other disabled widget.

I always have the impression the "Tip of the day" things don't really reach users that well...
And another "hey, you are new, check these links for help" pop-up is rather disruptive and annoying than helpful.

**Comment by Daenyth** [comment]
yeah, I don't think a popup, but maybe rather a default screen that just shows up on app load. Maybe embed buttons to the registration process too, or server selector

**Comment by tooomm** [comment]
>maybe a default screen that just shows up on app load. Maybe embed buttons to the registration process too, or server selector

A new tab could work:
![tab](https://user-images.githubusercontent.com/9874850/122175268-ccfb7000-ce83-11eb-9f95-ea5a6f387420.png)
